src/Tronkell/Server/Server.hs

```diff
@@ -42,8 +42,13 @@ startServer gConfig = do
   -- start network
   wsThread  <- forkIO $ W.start firstUId networkChans clientsChan
   tcpThread <- forkIO $ Tcp.start firstUId networkChans clientsChan
+  -- to avoid space msg leak..
+  forkIO . forever . readChan $ clientSpecificOutChan
+  forkIO . forever . readChan $ clientsChan
+  forkIO . forever . readChan $ internalChan
```

Comment:
i tried creating a `broadcast fn`.. but writes are to `duplicate chan` and at that point we do/may not have access to main channel.. 
i am adding a fixme for now.